politeiad/backendv2/tstorebe/plugins/pi/cmds.go

```diff
@@ -242,25 +242,131 @@ func (p *piPlugin) cmdBillingStatusChanges(token []byte) (string, error) {
 	return string(reply), nil
 }
 
+// cacheDataGet retrieves the data associated with the given token from the
+// memory cache.  If data doesn't exist in cache it returns nil.
+func (p *piPlugin) cacheDataGet(token string) *cacheData {
```

Comment:
These should not be piPlugin methods.  They should be methods on the cache struct.